charts/cinder-csi-plugin/values.yaml

```diff
@@ -82,3 +82,42 @@ storageClass:
   retain:
     isDefault: false
     allowVolumeExpansion: true
+# any kind of custom StorageClasses
+#   custom: |-
+#     ---
+#     apiVersion: storage.k8s.io/v1
+#     kind: StorageClass
+#     metadata:
+#       name: sas
+#     provisioner: cinder.csi.openstack.org
+#     volumeBindingMode: WaitForFirstConsumer
+#     allowVolumeExpansion: true
+#     parameters:
+#       type: SAS
+#     ---
+#     apiVersion: storage.k8s.io/v1
+#     kind: StorageClass
+#     metadata:
+#       name: ssd
+#     provisioner: cinder.csi.openstack.org
+#     volumeBindingMode: WaitForFirstConsumer
+#     allowVolumeExpansion: true
+#     parameters:
+#       type: SSD
+#     ---
+#     apiVersion: storage.k8s.io/v1
+#     kind: StorageClass
+#     metadata:
+#       name: sata
+#     provisioner: cinder.csi.openstack.org
+#     volumeBindingMode: WaitForFirstConsumer
+#     allowVolumeExpansion: true
+#     parameters:
+#       type: SATA
+#     ---
+#     apiVersion: snapshot.storage.k8s.io/v1beta1
+#     kind: VolumeSnapshotClass
+#     metadata:
+#       name: csi-cinder-snapclass
+#     driver: cinder.csi.openstack.org
+#     deletionPolicy: Delete
```

Comment:
I would prefer to only keep one example.

Reply:
I have now two different, so the user got an idea.